Examples/ExamplesApp/ExampleView.swift

```diff
@@ -20,6 +20,7 @@ struct ExampleView: View {
     var body: some View {
         example.makeBody()
             .navigationTitle(example.name)
+            .navigationBarTitleDisplayMode(.inline)
```

Comment:
This made a difference when building with Xcode 14.0 for Mac Catalyst. Based on that, I'm guessing it would make a difference for a Mac Catalyst app deployed to macOS 12.x.